src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,10 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
According to the checklist, you should never want to expose the object fields directly. They should be accessed through special methods (getters and setters). Please add getter and setter methods for the `maxCapacity` field.